hw2/yanagiragi/q1/.gitignore

```diff
@@ -0,0 +1,2 @@
+a.out
```

Comment:
no this

Reply:
Hi, so the main problem is that I shouldn't add any gitignore files? (e.g. hw2/yanagiragi/q1/.gitignore  and hw2/yanagiragi/q2/.gitignore )

If that's it, I apologize. I thought it won't be an issue since it only affects my folders. Let me know it there is any other problem.

Reply:
Also, I've notice the branch has been merged. I'm confused about should I patch it or not (the mail says I should). Please let me know what I should done to fix this. Thanks

Reply:
it's okay for this time. I just notice you that pushing the needed files.
No patch, since you don't have `invalied` label :)

Reply:
OK. I will fix it next time. One more thing, does that mean it is OK to ignore the last mail I received from E3?

Reply:
Yeah. You can dismiss that.